breakpad-symbols/src/lib.rs

```diff
@@ -895,10 +895,10 @@ mod test {
 
         let supplier = SimpleSymbolSupplier::new(paths.clone());
         let bad = SimpleModule::default();
-        assert_eq!(
-            supplier.locate_symbols(&bad).await,
-            Err(SymbolError::NotFound)
-        );
+        match supplier.locate_symbols(&bad).await {
+            Err(SymbolError::NotFound) => {}
+            _ => panic!(),
+        }
```

Comment:
these might be clearer as `assert!(matches!(expr, Err(SymbolError::NotFound))`

Reply:
Oh I didn't realize `matches!` was a thing! All of its friends (e.g. assert_matches) seemed to be unstable so I didn't look further.